deploy/0060_deploy_gems.ts

```diff
@@ -6,13 +6,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {deployments, getNamedAccounts} = hre;
   const {deploy} = deployments;
 
-  const {deployer} = await getNamedAccounts();
-  for (const gemName of gems) {
-    await deploy(`Gem_${gemName}`, {
-      contract: 'ERC20Token',
-      from: deployer,
+  const {gemOwner} = await getNamedAccounts();
```

Comment:
it should be `deployer` that deploy
sorry for the confusion with my comments on the tests
My comment was about other roles
teh `deployer` should be use to deploy but it should not be used for other roles as you run the risk of habing falsely correct test if you reuse the same address for different purpose